server/internal/usecase/interactor/project_test.go

```diff
@@ -497,6 +497,27 @@ func TestProject_Update(t *testing.T) {
 				MustBuild(),
 			wantErr: nil,
 		},
+		{
+			name:  "update visibility change",
```

Comment:
Add a new test case that covers the case of changing visibility will exceed the limit